geometry_msgs/PoseStamped[] points
bool flag
---
bool success